package.json

```diff
@@ -8,7 +8,9 @@
     "@testing-library/user-event": "^12.1.10",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-redux": "^7.2.2",
     "react-scripts": "4.0.2",
+    "redux": "^4.0.5",
```

Comment:
Opps!
You need to add prop-types for typechecking props. 

![image](https://user-images.githubusercontent.com/31897434/107975931-1e205380-6fb9-11eb-9edd-d93f3460e686.png)